packages/pyright-internal/src/analyzer/docStringConversion.ts

```diff
@@ -431,7 +436,8 @@ class DocStringConverter {
     }
 
     private _parseBacktickBlock(): void {
-        if (this._currentLine().startsWith('```')) {
+        // only match closing ``` at same indent level of opening
```

Comment:
Nit: Comments within the pyright code base begin with capital letters and end in a period if they are complete sentences.